frontend/src/metabase/admin/databases/components/NewDatabasePermissionsModal.tsx

```diff
@@ -1,39 +0,0 @@
-import { Link } from "react-router";
```

Comment:
@npfitz do you know if we have to do sth when removing translation strings or is all handled by CI jobs?